src/lib/input/input-container.ts

```diff
@@ -287,8 +315,14 @@ export class MdInputContainer implements AfterContentInit {
 
   @ContentChild(MdPlaceholder) _placeholderChild: MdPlaceholder;
 
+  @ContentChildren(MdErrorDirective) _errorChildren: QueryList<MdErrorDirective>;
+
   @ContentChildren(MdHint) _hintChildren: QueryList<MdHint>;
 
+  constructor(
+    private _changeDetectorRef: ChangeDetectorRef,
+    @Optional() private _parentForm: NgForm) { }
```

Comment:
If you inject `NgForm`, this will only work for template-driven forms (`formControl` only happens to work). Any form tag that has a `[formGroup]` directive on it becomes a `FormGroupDirective` instead, so this won't pick it up.